docs/modeling-your-data/modeling-your-data-with-dbt/dbt-ecommerce-data-model/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Ecommerce data model"
+date: "2022-11-17"
+sidebar_position: 104
+---
+
+# Snowplow Ecommerce Package
+
+**The package source code can be found in the [snowplow/dbt-snowplow-ecommerce repo](https://github.com/snowplow/dbt-snowplow-ecommerce), and the docs for the [model design here](https://snowplow.github.io/dbt-snowplow-ecommerce/#!/overview/snowplow_ecommerce).**
+
+The package contains a fully incremental model that transforms raw web ecommerce event data generated by the [Snowplow JavaScript tracker](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-trackers/) into a set of derived tables based around the following ecommerce data objects: carts, checkouts, products and transactions.
+
+The Snowplow ecommerce data model aggregates Snowplow's out of the box `snowplow_ecommerce_action` events to create a set of derived tables based on the ecommerce data objects - carts, checkouts, products, transactions, users - that contain many useful dimensions as well as calculated measures.
+
+## Overview
+
+The package contains multiple staging models however the mart models are as follows:
+
+- Base: Performs the incremental logic, outputting the table `snowplow_ecommerce_base_events_this_run` which contains a de-duped data set of all events required for the current run of the model, and is the foundation for all other models generated.
+- Carts: Parses the cart interactions that occur to provide handy filters and aggregations, and provides a Cart Sessions table which helps identify what happened to carts on a session level to extract, for example, abandoned carts with ease.
+- Checkouts: Parses checkout steps that occur to provide handy filters and aggregations to help identify which checkout steps were walked through, and what details were entered in each of these steps. This lends itself well to a funnel analysis
```

Comment:
❓ Do we want to link to the page on funnel analysis here or is it too different? https://docs.snowplow.io/docs/recipes/recipe-funnel-analysis/

Reply:
⛏️ Also, add `.`

Reply:
I think while the principle is the same, it just ends up being confusing because they're using a different context and then processing the data in a slightly different way. We will go through funnel analysis in the accelerator docs and can then even update the funnel analysis article to be pointing to these ecom schemas instead of the funnel schema